source/agora/common/Config.d

```diff
@@ -130,6 +130,9 @@ public struct NodeConfig
 
     /// Path to the data directory to store metadata and blockchain data
     public string data_dir = "/var/lib/agora/";
+
+    /// The minimum number of validators
+    public size_t min_validators = 0;
```

Comment:
This would mean that nodes which have a different minimum set in the config would fork.

The minimum number of validators needs to be a protocol constant.

Reply:
Okay, I'll try to use `agora.test.GenesisBlock.` I found this a little late.